HW1_data_explore.md

```diff
@@ -0,0 +1,28 @@
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+## HW1 Data Exploration
+
+In this assignment, I'll be exploring the Palmer Penguins data set!
+
+```{r penguins}
+HW1data<-palmerpenguins::penguins
+View(HW1data)
+```
+
+## Viewing Data Structure
+
+```{r penguins structure}
+str(HW1data)
+nrow(HW1data)
+ncol(HW1data)
+plot(HW1data$species) 
+mean(HW1data$bill_length_mm, na.rm=TRUE)
+```
```

Comment:
not sure why but I don't think the output of your code is not viewable in the "rich diff" format